test/e2e/multihoming_utils.go

```diff
@@ -83,6 +84,9 @@ func uniqueNadName(originalNetName string) string {
 }
 
 func generateNAD(config networkAttachmentConfig) *nadapi.NetworkAttachmentDefinition {
+	if config.mtu == 0 {
+		config.mtu = 1300
```

Comment:
It do not change the behaviour, there isn't any test passing the mtu except this new one since it's new field so for all the test is going to be 1300 as expected and we are also adding the value that you can now configure mtu at tests.

Reply:
Yes, but now you explicitly set the MTU to 1300 in *all* tests. Why can't you explicitly set the MTU in the *only* test that requires it ? 

Let's say for whatever reason the platform changes and the default provided by ovn-k stops being 1300. Then your code has changed the behavior of the tests.

Reply:
> Yes, but now you explicitly set the MTU to 1300 in _all_ tests. Why can't you explicitly set the MTU in the _only_ test that requires it ?
> 

It was 1300 explicity before, so I think it's the same
```diff
-        "mtu": 1300,
```

> Let's say for whatever reason the platform changes and the default provided by ovn-k stops being 1300. Then your code has changed the behavior of the tests.

As said the original version was explicitly seting it to 1300 is not something at ovn-k is at this very function.